src/components/Common/Link.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback, useMemo } from 'react'
+
+import classNames from 'classnames'
+
+import { isLocalLink, navigate } from '../../utils/locations'
+
+import './Link.css'
+
+export type LinkProps = React.AnchorHTMLAttributes<HTMLAnchorElement>
+
+export default React.memo(function Link({ target, rel, href, onClick, ...props }: LinkProps) {
+  const isLocal = useMemo(() => isLocalLink(href), [href])
+  const linkTarget = useMemo(() => (!target && !isLocal ? '_blank' : target || undefined), [isLocal, target])
+  const linkRel = useMemo(() => (!isLocal ? classNames(rel, 'noopener', 'noreferrer') : rel), [isLocal, rel])
```

Comment:
Are we passing a different type of `rel`? If we are not then I'd change this to `const linkRel = !isLocal ? "noopener noreferrer" : undefined`

Reply:
Here

![image](https://github.com/decentraland/governance/assets/45410089/b32c2eab-69d8-4b04-9ab6-6f85e533a967)